src/styles/blocks/benefits.scss

```diff
@@ -0,0 +1,42 @@
+.benefits {
+  // margin-top: 100px;
+
+  &__item {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+
+    transition: 0.5s;
```

Comment:
Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties.